shinysdr/plugins/rtty/__init__.py

```diff
@@ -55,59 +56,105 @@
 
 
 class RTTYDemodulator(gr.hier_block2, ExportedState):
+    '''Demodulate typical amateur RTTY.
+
+    Assumptions:
+
+        - 45.45 baud
+        - 170 Hz spacing
+        - Mark tone high
+
+    TODO: make these assumptions parameters.
+    '''
+
     implements(IDemodulator)
     
-    __filter_low = 1500
-    __filter_high = 2500
-    __transition = 100
+    _spacing = 170
+
+    _low_cutoff = _spacing * -2
+    _high_cutoff = _spacing
+    _transition_width = _spacing
 
-    def __init__(self, mode,
-            input_rate=0,
-            context=None):
+    def __init__(self, mode, input_rate=0, context=None):
         assert input_rate > 0
+        self.input_rate = input_rate
```

Comment:
don't make public attributes unless you have a reason for them to be public — use `__` prefix

(Yes, I know this contradicts standard Python style.)